### PR TITLE
Use old municipality names from municipality unions as synonyms

### PR DESCRIPTION
**Pull request by Joonalai**
This proposal adds old municipality names from municipality unions as synonyms for the regional data.

### CODE REVIEW

**Comment by Joonalai** [comment]
I added also the Swedish region names to English region synonym names, so that one can search with the Swedish name of the region as well.

File: pta-intelligent-search-api/src/test/resources/fi/maanmittauslaitos/pta/search/api/region/test_municipalities.json [hunk 1]
```diff
@@ -55,9 +55,15 @@
...
         "nimi": "Helsinki",
         "namn": "Helsingfors",
-        "synonyms_fi": ["Stadi"],
-        "synonyms_sv": ["Stad"],
-        "synonyms_en": ["City"],
+        "synonyms_fi": [
+          "Stadi"
+        ],
+        "synonyms_sv": [
+          "Stad"
```
Comment (sampov2) [inline pta-intelligent-search-api/src/test/resources/fi/maanmittauslaitos/pta/search/api/region/test_municipalities.json]: This is dangerous. A search term might have "borgå stad"

File: pta-intelligent-search-api/src/test/resources/fi/maanmittauslaitos/pta/search/api/region/test_municipalities.json [hunk 2]
```diff
@@ -55,9 +55,15 @@
...
+        ],
+        "synonyms_en": [
+          "City"
```
Comment (sampov2) [inline pta-intelligent-search-api/src/test/resources/fi/maanmittauslaitos/pta/search/api/region/test_municipalities.json]: Same as above. "City of Tampere" would/could map to Helsinki

Comment (Joonalai) [reply pta-intelligent-search-api/src/test/resources/fi/maanmittauslaitos/pta/search/api/region/test_municipalities.json]: This is just for testing purposes, but I'll remove these as well.

**Pull request merged**